test/integration/lib/render.js

```diff
@@ -49,22 +52,48 @@ tape.onFinish(() => {
     mapboxgl.clearPrewarmedResources();
 });
 
+let osIgnore;
+let timeout = 5000;
+
+if (process.env.CI) {
+    // On CI, MacOS and Windows run on virtual machines.
+    // These run slower so we increase the timeout.
+    const os = navigator.appVersion;
+    if (os.includes("Macintosh")) {
+        osIgnore = ignoreMac;
+        timeout = 20000;
+    } else if (os.includes("Linux")) {
+        osIgnore = null;
+    } else if (os.includes("Windows")) {
+        osIgnore = ignoreWindows;
+        timeout = 60000;
+    } else { console.warn("Unrecognized OS:", os); }
```

Comment:
Should we add:
```suggestion
    } else if (os.includes("Firefox")) {
        osIgnore = null;
    } else { console.warn("Unrecognized OS:", os); }
```
 So this doesn't warn on Firefox, or is this code path not enabled for this one?

Reply:
Explained below